installer/scripts/omsadmin.sh

```diff
@@ -813,6 +813,9 @@ copy_omsagent_d_conf()
 
     update_path $OMSAGENTD_DIR/monitor.conf
     update_path $OMSAGENTD_DIR/heartbeat.conf
+    if [ -s $OMSAGENTD_DIR/container.conf ] ; then
```

Comment:
Any reason you're using -s instead of -f? The rest of the file uses -f to check.